src/Core/Components/Dialog/FluentDialog.razor.css

```diff
@@ -3,12 +3,13 @@
     position: relative;
     padding: 0px var(--dialog-padding);
     width: calc(var(--dialog-width) - 2 * var(--dialog-padding));
-    height: calc(var(--dialog-height) - 2 * var(--dialog-padding));
-    max-width: 80%;
 }
```

Comment:
Why did you remove `max-width` ?

Reply:
Because on mobile I want a dialog or panel to take the whole width of the screen. Needs more changes probably

Reply:
In this case, we need to add a `@media` style to remove the `max-width` on mobile.
And I'm not even sure it's a good idea to accept 100% on mobile. Because a DialogBox should look like a window, even on mobile (in my opinion).